scripts/esb_sniffer/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _esb_sniffer_scripts_docs:
+
+ESB Sniffer
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+This directory contains python scripts for the :ref:`esb_monitor` sample configured as ESB Sniffer.
+
+Setup Linux
+***********
+
+1. Install python requirements:
+
+   .. parsed-literal::
+         :class: highlight
+
+         pip3 install -r requirements.txt
+
+#. Add custom dissector and extcap to the Wireshark:
+
+   .. parsed-literal::
+         :class: highlight
+
+         mkdir -p $HOME/.local/lib/wireshark/{extcap,plugins}
+         cp esb_dissector.lua $HOME/.local/lib/wireshark/plugins
+         cp extcap/esb_extcap.py $HOME/.local/lib/wireshark/extcap
+         chmod +x $HOME/.local/lib/wireshark/extcap/esb_extcap.py
+
+#. Enable dissector for ESB:
+
+   * Open Wireshark
+   * Go to Edit -> Preferences -> Protocols -> DLT_USER -> Edit
+   * Click Create new entry icon (bottom left)
+   * Select *DLT=147* for DLT column and *esb* for Payload dissector column
+   * Click Ok
+   * Restart Wireshark
+
+#. After completing these steps, a new ESB sniffer interface should appear in Wireshark.
+
+Scripts
+*******
+
```

Comment:
A heading after another is not good practice. Have some introductory text here. Something like: There are two separate scripts you need for configuring the ESB sniffer.